packages/react-app/src/constants.js

```diff
@@ -9,6 +9,8 @@ export const BLOCKNATIVE_DAPPID = "0b58206a-f3c0-4701-a62f-73c7243e8c77";
 
 export const ALCHEMY_KEY = "oKxs-03sij-U_N0iOlrSsZFr29-IqbuF";
 
+export const RPC_POLL_TIME = 3000;
```

Comment:
Maybe 5000 as default?

Also as add a comment above telling what that constant is doing.

have you tried with 0? That should do RPC request on block, as it was before.